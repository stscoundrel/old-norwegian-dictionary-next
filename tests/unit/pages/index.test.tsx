import ReactDOM from 'react-dom'
import Index from 'pages/index'
import renderer from 'react-test-renderer'

describe('Index page', () => {
  test('Does not crash', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Index />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('Matches snapshot', () => {
    const tree = renderer.create(<Index />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
