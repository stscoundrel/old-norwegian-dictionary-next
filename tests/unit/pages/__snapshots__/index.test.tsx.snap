// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Matches snapshot 1`] = `
Array [
  <main
    className="container"
  >
    <section
      className="section"
    >
      <p>
        There is no strength in numbers
      </p>
      <p>
        Have no such misconception
      </p>
      <p>
        But when you need me be assured I wont be far away
      </p>
    </section>
  </main>,
  <footer
    className="section"
  >
    <div
      className="container"
    >
      <p>
        Copyright Â© 
        2021
      </p>
    </div>
  </footer>,
]
`;
